tracer/src/Datadog.Trace/Debugger/Snapshots/DebuggerSnapshotSerializer.UnreachableLocal.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="DebuggerSnapshotSerializer.UnreachableLocal.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace.Debugger.Snapshots
+{
+    internal static partial class DebuggerSnapshotSerializer
+    {
+        /// <summary>
+        /// This class hold a local variable inside an async method that was not hoisted by
+        /// the state machine object but kept as regular local. Hence we can not get his value
+        /// at method end and the value will be always default.
```

Comment:
```suggestion
        /// at method end and the value will always the the default value.
```

Reply:
Fixed in 6362e7d75721c66e43ffa8d5e0da1e9df7daf7bc